test/e2e_tests/configmap.go

```diff
@@ -0,0 +1,71 @@
+package e2e_tests
+
+import (
+	"fmt"
+	"github.com/dana-team/platform-backend/src/types"
+	"github.com/dana-team/platform-backend/src/utils/testutils"
+	"github.com/dana-team/platform-backend/src/utils/testutils/mocks"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"net/http"
+)
+
+var _ = Describe("Validate ConfigMap routes and functionality", func() {
+	var namespaceName, configMapName string
+	var namespace corev1.Namespace
+
+	BeforeEach(func() {
+		namespaceName = generateName(e2eNamespace)
+		namespace = mocks.PrepareNamespace(namespaceName, map[string]string{e2eLabelKey: e2eLabelValue})
```

Comment:
ill create a function to create e2e namespace because it so common

Reply:
Created similar functions for other resources as well